src/main/kotlin/venus/Renderer.kt

```diff
@@ -244,20 +248,30 @@ internal object Renderer {
         newRow.id = "instruction-$pcx"
         newRow.onclick = { Driver.addBreakpoint(pcx) }
 
-        val pcline = newRow.insertCell(0)
+        var cellIndex = 0
+        val breakpointLine = newRow.insertCell(cellIndex)
+        val breakpointNode = document.createElement("div")
+        breakpointNode.setAttribute("name", "breakpoint_cell")
+        breakpointLine.appendChild(breakpointNode)
+        cellIndex += 1

```

Comment:
`++` please, I think this repo mostly uses it (same with line 261)

Reply:
Done